unit-tests/dds/test-librs-logic.py

```diff
@@ -0,0 +1,90 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Lets rename the file to a more meaningful name
`test-discovery`?
`test-dev-connection`?

Reply:
The original intent was to test all kind of logics here, in addition to the device properties testes in test-librs-device-properties.py. In order to avoid dependencies between tests it will probably be better to split into multiple files.

I will rename test-librs-connections.py